splash.py

```diff
@@ -11,43 +11,57 @@
 import apt
 import threading
 import requests
+from pathlib import Path
+
+global home
+home = str(Path.home())
+print(f"{home} is your home directory!")
+
 
 class aptFind(apt.Cache):
     def __init__(self):
         super().__init__()
+
     def find(self, pkg):
         try:
             if self[pkg].is_installed:
-                print(pkg+" is installed")
+                print(pkg + " is installed")
                 return True
             else:
-                print(pkg+" is NOT installed")
+                print(pkg + " is NOT installed")
                 return False
         except KeyError:
-            print("ERROR: there is no package called '"+pkg+"'!")
+            print("ERROR: there is no package called '" + pkg + "'!")
+
     def install(self, pkg):
-        command='sudo apt install -y '+pkg
+        command = "sudo apt install -y " + pkg
         cmd(command)
+
     def installIfNotInstalled(self, pkg):
         if not self.find(pkg):
             self.install(pkg)
+
+
 def pip_install(package):
     subprocess.check_call([sys.executable, "-m", "pip", "install", package])
 
+
 def install_depends():
     global isInstalling
-    isInstalling=True
+    isInstalling = True
     c = aptFind()
-    c.installIfNotInstalled('xterm')
-    c.installIfNotInstalled('python3-pil')
-    c.installIfNotInstalled('python3-pil.imagetk')
-    c.installIfNotInstalled('python3-pip')
-    c.installIfNotInstalled('mpg123')
-    c.installIfNotInstalled('lolcat')
-    pip_install('distro')
-    pip_install('psutil')
-    #pip_install('pywebview')
-    isInstalling=False
+    c.installIfNotInstalled("xterm")
+    c.installIfNotInstalled("python3-pil")
+    c.installIfNotInstalled("python3-pil.imagetk")
+    c.installIfNotInstalled("python3-pip")
+    c.installIfNotInstalled("mpg123")
+    c.installIfNotInstalled("lolcat")
```

Comment:
Could be replaced with
```python
for pkg in ["xterm", "python3-pil", "python3-pil.imagetk", "python3-pip", "mpg123", "lolcat"]:
    c.installIfNotInstalled(pkg)
```

Reply:
I'll add it to the next update. This update was really exhausting to write. I need a little break first xD

Reply:
You definitely deserve a break!